docs/source/tutorial.rst

```diff
@@ -0,0 +1,65 @@
+Tutorial
```

Comment:
love this, just laser-targeted at the primary use case

Reply:
Thanks!